scripts/calibrate_vcs.py

```diff
@@ -653,16 +653,20 @@ def write_cont_scripts(self):
                         "nodes": "{0}".format(nnodes),
                         "gres": "gpu:1",
                         "ntasks-per-node": "1"}
-        module_list = ["RTS/master"]
+        module_list = ["RTS/chj"]
```

Comment:
Is this a permanent change to the module we load or will it eventually be changed back to master?

Reply:
I assume so but I've messaged Chris to double check

Reply:
I talked to Chris and he's installed the master version for us